disk-buffering/src/main/java/io/opentelemetry/contrib/disk/buffering/internal/serialization/deserializers/MetricDataDeerializer.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.disk.buffering.internal.serialization.deserializers;
+
+import io.opentelemetry.contrib.disk.buffering.internal.serialization.mapping.metrics.ProtoMetricsDataMapper;
+import io.opentelemetry.proto.metrics.v1.MetricsData;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import java.io.IOException;
+import java.util.List;
+
+public final class MetricDataDeerializer implements SignalDeserializer<MetricData> {
```

Comment:
```suggestion
public final class MetricDataDeserializer implements SignalDeserializer<MetricData> {
```

Reply:
Whoopsie! Thanks for the catch....will fix.